### ISSUE TITLE
LED4 should be labled "75%" and should be green

### ISSUE THREAD

**Issue by jadonk** [beagleboard/robotics-cape#27]
This led is used at the battery 75% full indicator.

Ideally, there would be 50% and 100% indicators as well, but we seem to be out of I/O pins.

See https://github.com/beagleboard/capes/tree/master/beaglebone/Robotics that this cape replaces.

**Comment by lorforlinux** [comment]
@jadonk does that means we are skipping the 50% battery indicator to use one of the LEDs as use LED?
Asking because you mentioned to use LED3 as 25% in this issue -> https://github.com/beagleboard/robotics-cape/issues/26

**Comment by jadonk** [comment]
Yes, simply due to a lack of LEDs. Your idea of charlieplexing is good, but
we should do a PRU firmware load for that and an associated leds driver.
